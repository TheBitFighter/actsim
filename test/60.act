defproc test()
{
  int<8> x;
   
  chp {
   [ {x{1..0},x{2..1}} = x -> log ("equal") [] else -> skip ]
  }
}


