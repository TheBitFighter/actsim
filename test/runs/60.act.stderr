In expanding test<> (60.act:1)
WARNING: Process `test<>': chp local variable `x': no driver
WARNING: test<>: substituting chp model (requested prs, not found)
